--- /mnt/projects/dpdk/drivers/net/Makefile	2015-12-18 11:43:46.811004600 +0800
+++ /home/jenkinzh/dpdk-2.2.0/drivers/net/Makefile	2016-02-13 11:16:32.717359308 +0800
@@ -52,5 +52,6 @@
 DIRS-$(CONFIG_RTE_LIBRTE_VMXNET3_PMD) += vmxnet3
 DIRS-$(CONFIG_RTE_LIBRTE_PMD_XENVIRT) += xenvirt
 
+DIRS-$(CONFIG_RTE_LIBRTE_PMD_VLINKY) += vlinky
 include $(RTE_SDK)/mk/rte.sharelib.mk
 include $(RTE_SDK)/mk/rte.subdir.mk
